### DISCUSSION TITLE
Long division

### DISCUSSION THREAD

**Discussion by StevenClontz** [TeamBasedInquiryLearning/precalculus#214] [Ideas]
I will return to this thread to catch the internet up on what was discussed in emails between me, @siwelwerd and @sheppardw , but don't have time until later today.

**Comment by siwelwerd** [comment]
[This post](https://tex.stackexchange.com/questions/131125/better-way-to-display-long-division) suggests using `\smash` and some other small tweaks to improve the appearance.

**Comment by StevenClontz** [comment]
Relevant PR: https://github.com/TeamBasedInquiryLearning/precalculus/pull/213

Question: How can we put examples of long division in a PreTeXt document that works in both web and print?

**Comment by StevenClontz** [comment]
Okay, the tl;dr here is that we need a pattern for long division. This turns out to be famously difficult to do in a way that's supported both by LaTeX (used to create our PDFs) and MathJaX (powering our HTML).

I think the best solution for now is to create long division examples as images. These are super inaccessible to screenreader users, so we should consider how to accommodate such learners, but for now here's a solution that works for visual readers in both HTML and print: (h/t to Drew for fixing my original bug and suggesting `/smash` to make things prettier):

```xml
   <activity>
          <statement>
            <p>Long division problem here with blanks to fill in</p>
              <image xml:id="example-long-division" width="25%">
                <latex-image>
                  \(
                \begin{array}{ r *{19}{c@{}} }
                &amp; &amp; 0 &amp; . &amp; 7 &amp; 1 &amp; 6 &amp; 2 \\ \cline{2-9}\\[-14.5pt]
                \multicolumn{1}{r}{74\rlap{\kern4.5pt\smash{\raise.2ex\hbox{)}}}} &amp; 5 &amp; 3 &amp; . &amp; 0 &amp; 0 &amp; 0 &amp; 0 &amp; 0 \\
                &amp; 5 &amp; 1 &amp; &amp; 8 \\ \cline{2-5}
                &amp; &amp; 1 &amp; &amp; 2 &amp; 0 \\
                &amp; &amp; &amp; &amp; 7 &amp; 4 \\  \cline{4-6}
                &amp; &amp; &amp; &amp; 4 &amp; 6 &amp; 0 \\
                &amp; &amp; &amp; &amp; 4 &amp; 4 &amp; 4 \\   \cline{5-7}
                &amp; &amp; &amp; &amp; &amp; 1 &amp; 6 &amp; 0 \\
                &amp; &amp; &amp; &amp; &amp; 1 &amp; 4 &amp; 8 \\  \cline{6-8}
                &amp; &amp; &amp; &amp; &amp; &amp; 1 &amp; 2 &amp; 0
                \end{array}
                  \)
                </latex-image>
                </image>
              </statement>
              </activity>
```

HTML:

![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/1559632/b13c3cd2-6d1a-429d-82f8-fea35ee51358)

PDF:

![image](https://github.com/TeamBasedInquiryLearning/precalculus/assets/1559632/e6a4aff9-94a7-44a3-b7d3-d13fd3fa41fb)

**Comment by StevenClontz** [comment]
@sheppardw if you want to write on paper exactly what you want your long division in this activity to look like, I'll code up at least the first example for you. For example, you want blanks left in the calculation, correct? And maybe you really want polynomial long division?
